docs/docs/content-and-data.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: Sourcing Content and Data
+title: Sourcing Content & Data
```

Comment:
```suggestion
title: Sourcing Content and Data
```
"and" is spelled out in the URL, so I'd keep this one as-is.

Reply:
Thanks for the comments - I can make the changes and submit a new PR to better adhere to the preferred style.